src/CMakeLists.txt

```diff
@@ -550,80 +678,34 @@ set_target_properties(${src_executable_gtests}
 add_custom_target(tests_unit
                   COMMAND ${src_executable_gtests}
                   DEPENDS ${src_executable_gtests}
+                  COMMENT "Executing test ${src_executable_gtests}"
                   VERBATIM)
-add_dependencies(${src_executable_gtests} ${src_lib_static_gtest} ${src_common_test_exe_libs})
 
 #
 # tests_all just calls other targets
 #
+# TODO This doesn't seem to have any effect; it's probably because the DEPENDS
```

Comment:
@scottpurdy: running of the tests is (and has been) broken. We actually run the tests after the build completes. Do we really want to run tests during `make`? If not, we should remove this and the underlying custom targets